index.html

```diff
@@ -0,0 +1,200 @@
+                <hr class="horizontal-line">
+            </section>
+            <div class="more">
+            <ul class="about-list">
+                <ul class="land">
+                <li class="language">Language</li>
+                <li class="javascript">javascript</li>
+                <li class="ruby">Ruby</li>
+                <li class="li-html">Html</li>
+                <li class="li-css">Css</li>
+                </ul>
+                <ul class="fame">
+                <li class="frame">Frameworks</li>
+                <li class="bootstrap">Bootstrap</li>
+                <li class="ruby-on-rails">Ruby on Rails</li>
+                <li class="rspec">RSPEC</li>
+                <li class="capy">CapyBara</li>
+                <li class="selen">Selenium</li>
+                </ul>
+                <ul class="skil">
+                <li class="skills">Skills</li>
+                <li class="codekit">Codekit</li>
+                <li class="github">Github</li>
+                <li class="codepen">Codepen</li>
+                <li class="gitlab">Gitlab</li>
+                <li class="terminal">Terminal</li>
+                </ul>
+            </ul>
+            </div>
+        </div> 
+        <!-- work section ends       -->
+
+        <div class="form-start">
+            <h2 class="form-header">I'm always interested in hearing about your new projects, so if you'd like to chat please get in</h2>
+            <form action="https://formspree.io/f/xlekbqne" method="post">
+                <fieldset>
+              <label for="full-name"></label>
+              <input type="text" id="full-name" name="full-name" maxlength="30" placeholder="Full name" required>
+              <input type="text" id="First-name" name="first-name" maxlength="" placeholder="First Name" required>
+              <input type="text" id="last-name" name="last-name" maxlength="" placeholder="Last Name" required>
+              <label for="email"></label>
+              <input type="email" id="email" name="email" placeholder="Email address" required>
+              <label for="message"></label>
+              <textarea id="message" name="message"  required>Enter text here</textarea>
+                    <button type="submit" class="submit">Get in touch</button>
+                </fieldset>
+            </form>
+                <hr class="seperator">
+                <div class="footer">
+                <ul class="base">
+                                    <li><a href="https://www.twitter.com/jiyapete/"><i class="fa-brands fa-twitter" id="twitte"></i></a></li>
+                                    <li><a href="https://www.linkedin.com/peteronyam/"><i class="fa-brands fa-linkedin" id="link"></i></a></li>
+                                    <li><a href="https://www.github.com/peteronyam"><i class="fa-brands fa-github" id="git"></i></a></li>
+                            <hr class="bar">
+                </ul>
+                </div>
+                </div>
```

Comment:
- [ ] As I mentioned above :point_up: please stick to the Figma design but your social media icons are not the same as the Figma design and also your icons are not aligned properly so kindly fix these issues also. Check below :point_down:  the comparison between your design and Fidma Design :+1: 

**Your Design**
![Screenshot from 2023-03-05 02-55-02](https://user-images.githubusercontent.com/45226848/222931592-72748d6b-c7d9-46ab-af99-b1ba27c39dc0.png)

**Figma design**
![Screenshot from 2023-03-05 03-30-35](https://user-images.githubusercontent.com/45226848/222931630-242347b7-75af-4da0-9966-afd4c30f80b1.png)